immersion/kubeflow_pipelines/pipelines/solutions/lab-02_vertex.ipynb

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    response = api_client.create_run_from_job_spec(job_spec_path=PIPELINE_JSON)
</pre><p>If the Vertex AI API is not enabled for the project, this results in an error. The error itself is informative in including an URL to enable the API, but optionally in the text we could mention that it should be enabled at https://console.developers.google.com/apis/api/aiplatform.googleapis.com/overview</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Replaced that with a call to the aiplatform sdk.</p>